Source/ee/SIF.cpp

```diff
@@ -291,7 +291,7 @@ void CSIF::SaveState(Framework::CZipArchiveWriter& archive)
 	{
 		CStructCollectionStateFile* callRepliesFile = new CStructCollectionStateFile(STATE_CALL_REPLIES_XML);
 		for(CallReplyMap::const_iterator callReplyIterator(m_callReplies.begin());
-			callReplyIterator != m_callReplies.end(); callReplyIterator++)
+			callReplyIterator != m_callReplies.end(); ++callReplyIterator)
```

Comment:
Should use a [range for loop](http://en.cppreference.com/w/cpp/language/range-for) instead.